src/utils/productUtils.ts

```diff
@@ -0,0 +1,33 @@
+import { Product } from '../types/interface';
+
+function isSportsWear(product: Product) {
+  const { productView } = product;
+  const department = productView?.attributes?.find(({ name }) => name === 'pim_department_name');
+  
+  return department?.value.includes('Sportswear');
+}
+
+function getColorSwatchesFromQuery(item: Product) {
+  if (isSportsWear(item)) {
+    return [];
+  }
+
+  const { productView } = item;
+  const attributeId = productView?.options?.[0].id;
+  const imageAttributes = productView?.attributes?.find(({ name }) => name === 'eds_images');
+  if (imageAttributes) {
```

Comment:
same as before, the requirements to select the images are quite confusing, so would be great if you can explain them with comments in the code